feature/signup/src/main/java/com/yapp/feature/signup/signup/SignUpViewModel.kt

```
@@ -0,0 +1,86 @@
+                        primaryButtonEnable = updatePrimaryButtonEnable(previousStep)
+                    )
+                }
+            }
+
+            SignUpIntent.ClickPrimaryButton -> {
+                val nextStep = when (state.currentStep) {
+                    SignUpStep.Name -> SignUpStep.Email
+                    SignUpStep.Email -> SignUpStep.Password
+                    SignUpStep.Password -> SignUpStep.Position
+                    SignUpStep.Position -> SignUpStep.Complete // TODO SignUpStep.Pending
+                    SignUpStep.Complete -> TODO()
```

detekt (1.23.7)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)